deploy/crds/csi.ibm.com_v1_ibmblockcsi_cr.yaml

```diff
@@ -3,10 +3,56 @@ kind: IBMBlockCSI
 metadata:
   name: ibm-block-csi
   namespace: kube-system
+  labels:
+    app.kubernetes.io/name: ibm-block-csi-operator
+    app.kubernetes.io/instance: ibm-block-csi-operator
+    app.kubernetes.io/managed-by: ibm-block-csi-operator
 spec:
   controller:
     repository: ibmcom/ibm-block-csi-driver-controller
     tag: "1.0.0"
+    imagePullPolicy: IfNotPresent
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+
   node:
     repository: ibmcom/ibm-block-csi-driver-node
     tag: "1.0.0"
+    imagePullPolicy: IfNotPresent
+    affinity:
+      nodeAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          nodeSelectorTerms:
+            - matchExpressions:
+                - key: kubernetes.io/arch
+                  operator: In
+                  values:
+                    - amd64
+
+  sidecars:
+  - name: csi-node-driver-registrar
+    repository: quay.io/k8scsi/csi-node-driver-registrar
+    tag: "v1.2.0"
+    imagePullPolicy: IfNotPresent
+  - name: csi-provisioner
+    repository: quay.io/k8scsi/csi-provisioner
+    tag: "v1.3.0"
+    imagePullPolicy: IfNotPresent
+  - name: csi-attacher
+    repository: quay.io/k8scsi/csi-attacher
+    tag: "v1.2.1"
+    imagePullPolicy: IfNotPresent
+  - name: livenessprobe
+    repository: quay.io/k8scsi/livenessprobe
+    tag: "v1.1.0"
+    imagePullPolicy: IfNotPresent
+
+  imagePullSecrets:
+  - "secret1"
```

Comment:
why you put secret1 ????
it should be in comment out  these 2 lines. just if needed then user will add it.
if u keep secret1 then user will have to delete it.

thoughs?

Reply:
ok, I will comment it out

Reply:
I do not see where the default serviceaccount (used by the node pod) would get updated with the imagePullSecret.  Is there a reason a serviceAccount isn't used by node?

Reply:
I create a new sa for node, I think it is not a good practice to use the default sa, because it is uncontrollable. Also, it is an icp warning.

Reply:
remember to comment imagePullSecrets it out and don't put "secret1" just secretkey.